config/database.yml

```diff
@@ -0,0 +1,87 @@
+# PostgreSQL. Versions 9.1 and up are supported.
+#
+# Install the pg driver:
```

Comment:
Same as before, not a correction, but about comments: I don't mind commented configuration lines but you can choose to remove them. I personally prefer clean configuration files (you can always add back those lines or read the official documentation for each gem) since it's more clear what you're actually using.